floater/generators.py

```diff
@@ -10,13 +10,14 @@
 
 
 def geo_to_xyz(geo_cord):
-    lon_deg, lat_deg = geo_cord.transpose()
+    lon_deg, lat_deg,depth =np.vsplit(geo_cord,3)
+    earthrad=6371000
     lat = np.radians(lat_deg)
     lon = np.radians(lon_deg)
-    x = np.cos(lat)*np.cos(lon)
-    y = np.cos(lat)*np.sin(lon)
-    z = np.sin(lat)
-    xyz_cord = np.transpose(np.array((x,y,z)))
+    x =  (earthrad+depth)*np.cos(lat)*np.cos(lon)
+    y =  (earthrad+depth)*np.cos(lat)*np.sin(lon)
+    z =  (earthrad+depth)*np.sin(lat)
+    xyz_cord = np.squeeze(np.transpose(np.array((x,y,z)),(2,0,1)))
     return xyz_cord
```

Comment:
Just a question about this... it looks like the z coordinate will come back as meters from the center of the earth. For MITgcm, we usually think about depth as meters from sea level. Do you anticipate any conflicts here?